packages/components/src/components/input-field/input.jsx

```diff
@@ -96,6 +102,7 @@ const Input = ({
                 type={type === 'number' ? 'text' : type}
                 value={display_value || ''}
                 aria-label={ariaLabel}
+                data-lpignore={type === 'password' ? undefined : true}
```

Comment:
```suggestion
                data-lpignore={type !== 'password'}
```
Simplified check. Ignore only if its not a password

Reply:
Will change it